_data-prepper/pipelines/configuration/processors/aggregate.md

```diff
@@ -20,8 +20,31 @@ group_duration | No | String | The amount of time that a group should exist befo
 
 <!---## Configuration
 
-Content will be added to this section.
+Content will be added to this section.--->
 
 ## Metrics
 
-Content will be added to this section.--->
\ No newline at end of file
+Apart from common metrics in the [Abstract processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java), the `aggregate` processor introduces the following custom metrics:
```

Comment:
I think we should outline each of the metrics which come from the "Abstract processor" rather than linking users.

Reply:
@carolxob , For each processor we should add:

| Metric Name | Type | Description |
| ------------- | ---- | -----------|
| `recordsIn` | Counter | Metric representing the ingress of records to a pipeline component. |
| `recordsOut` | Counter | Metric representing the egress of records from a pipeline component. |
| `timeElapsed` | Timer | Metric representing the time elapsed during execution of a pipeline component. |